reducers/plugins/index.js

```diff
@@ -224,4 +255,8 @@ export default combineReducers({
     // object where every key is a post type and the values are components wrapped in an
     // an object that contains a plugin id
     postCardTypes,
+
+    // object where every key is a post type and the values are components wrapped in an
```

Comment:
This documentations should be updated for the new added `adminConsoleReducers`

Reply:
Fixed